lib/components/SideNote/index.tsx

```diff
@@ -0,0 +1,62 @@
+import {
+  ComponentProps,
+  MouseEventHandler,
+  PropsWithChildren,
+  ReactElement,
+} from "react";
+import clsx from "clsx";
+import { ChevronRightIcon } from "./ChevronRightIcon";
+import { Text } from "../Text";
+import "./SideNote.scss";
+
+type SideNoteProps = Omit<ComponentProps<"div">, "className"> & {
+  title?: string | ReactElement;
```

Comment:
```suggestion
  title?: string | React.JSX.Element;
```

Reply:
can we use `ReactNode` instead. ReactNode includes `string` aswell 😃 .
```suggestion
  title?: ReactNode;
```